CDP4PluginInstaller/CDP4PluginInstaller.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
Did I undestand correctly that this project is not needed anymore? If yes please clean up.